.github/workflows/e2e.yaml

```diff
@@ -30,11 +30,34 @@ jobs:
           path: kmm_local.tar
           retention-days: 1
 
+  build-signing-image:
+    runs-on: ubuntu-latest
+
+    name: Build the signing image
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Build the image
+        run:  make signimage-build SIGNER_IMG=kmm-signimage
+
+      - name: Export the image
+        run: docker save -o kmm-signimage.tar kmm-signimage
+
+      - name: Upload the image
+        uses: actions/upload-artifact@v3
+        with:
+          name: ci-images
+          if-no-files-found: error
+          path: kmm-signimage.tar
```

Comment:
```suggestion
          path: kmm_signimage_local.tar
```

Reply:
Not sure if we want to append `local`, I initially used it to signal that it's inserted into the minikube nodes - not the case here.

Reply:
I think that "local" means implies that this image was build locally in CI (unlike using remote images unrelated to the changes in a PR) regardless of it is in the node registry or in `minikube`'s registry.

As long as it is built as part of the PR and not pushed anywhere remotely, I think we should call it `local` or else just remove the `local` from KMM as well - because there is no *logical* difference between the 2. It is just confusing IMO.